src/jit/importer.cpp

```diff
@@ -4989,6 +4989,25 @@ GenTreePtr Compiler::impImportLdvirtftn(GenTreePtr              thisPtr,
         NO_WAY("Virtual call to a function added via EnC is not supported");
     }
 
+#if COR_JIT_EE_VERSION > 460
```

Comment:
The CoreRT-specific handling of GVMs and fat functions pointers should be only under `CORERT_ABI`, and not under R2R.